includes/class-stats.php

```diff
@@ -658,26 +666,82 @@ public function get_order_item_refund_count( $query = array() ) {
 	 * @return string Formatted amount from refunded orders.
 	 */
 	public function get_order_refund_amount( $query = array() ) {
-		$query['status'] = array( 'complete' );
-		$query['type']   = array( 'refund' );
+		$this->parse_query( $query );
+
+		// Add table and column name to query_vars to assist with date query generation.
+		$this->query_vars['table']             = $this->get_db()->edd_orders;
+		$this->query_vars['date_query_column'] = 'date_created';
+
+		if ( empty( $this->query_vars['function'] ) ) {
+			$this->query_vars['function'] = 'SUM';
+		}
 
 		/*
-		 * Request raw output so we can run `abs()` on the value.
-		 * But save the original value so we can restore it later.
+		 * By default we're checking sales only and excluding refunds. This gives us gross order earnings.
```

Comment:
This comment is confusing me because the `type` is explicitly set to refund. Should the wording be inverted instead?